classes.js

```diff
@@ -0,0 +1,234 @@
+//         this.name = name
+//         this.breed = breed
+//         this.mood = mood
+//         this._hungry = _hungry
+
+//     }
+//     playFetch(){
+//         if(this._hungry === true) {
+//             this.mood = "Playful"
+//             return "RUFF!"
+//         }
+//     }
+//     feed(){
+//         if(this._hungry === true){
+//             this._hungry === false
+//             return "WHOOFF!"
+//         } else {
+//             this._hungry === false
+//             return "the dog does not look hungry"
+//         }
+//     }
+//     toString(){
+//         return this.name +" " +this.breed+" "+this.mood+" "+this._hungry
+//     }
+// }
+// let fido = new Dog("Fido", "Yorkie", "happy", false)
+// console.log(fido.toString())
+
+// // # Question 4
+// // There are three common scales that are used to measure temperature: Celsius, Fahrenheit, and Kelvin:
+// // C = (F - 32) / 1.8
+// // F = 1.8 * C + 32
+// // K = C + 273
+// // a. Make an object called `freezingPoint` that has three properties: `celsius`, `fahrenheit`, and `kelvin`. 
+// // Give them all values equal to the freezing point of water.
+// // b. Make a class called `Celsius` that has one property: `celsius`, and two methods `getFahrenheitTemp`, and `getKelvinTemp`.
+
+// // ```js
+// // let outsideTempt = new Celsius(10.0)
+// // outsideTempt.celsius //returns 10.0
+// // outsideTempt.getKelvinTemp() //returns 283.0
+// // outsideTempt.getFahrenheitTemp() //returns 50.0
+// // c. Give `Celsius` a method called `isBelowFreezing` that returns a `Bool` (true if the temperature is below freezing).
+
+// class freezingPoint {
+//     constructor(celsius,fahrenheit,kelvin){
+//         this.celsius = 0
+//         this.fahrenheit = 32
+//         this.kelvin = 273.2
+//     }
+// }
+// class Celsius {
+//     constructor(celsius){
+//         this.celsius = celsius
+//     }
+//     getFahrenheitTemp(){
+//         return 1.8 * this.celsius + 32
+//     }
+//     getKelvinTemp(){
+//         return this.celsius + 273
+//     }
+//     isBelowFreezing(){
+//         if(this.celsius <= 0){
+//         return true
+//     } else {
+//         return false
+//     } 
+//     }          
+// }
+// let outsideTemp = new Celsius(10.0)
+// console.log(outsideTemp.getKelvinTemp())
+
+// # Question 5
+// a. Create a class called `Movie` that has properties for `name`, `year`, `genre`, `cast`, and `description`. 
+// Create an instance of your `Movie`
+// b. Create an method inside `Movie` called `blurb` 
+// that returns a formatted string describing the movie.
+// Ex: "Borat came out in 2006. It was an odd film starring Sacha Baron Cohen as a man named Borat who was visiting 
+// America from Kazakhstan."
+
+// class Movie {
+//     constructor(name,year,genre,cast,description){
+//         this.name = name;
+//         this.year = year;
+//         this.genre = genre;
+//         this.cast = cast;
+//         this.description = description
+//     }
+//     blurb(){
```

Comment:
This is hardcoding a blurb. We want to be able to call blurb on any movie object, and not just hear about Shawshank (although it IS an excellent movie)